app/src/main/res/values-lb/strings.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <string name="app_name">Keen Hond</string>
+    <string name="dog">Hond</string>
+    <string name="action_settings">Astellungen</string>
+    <string name="permission_explanation">Mir kennen den Hond leider net gesinn wann's du eis net erlaabs d'Kamera ze benotzen.</string>
```

Comment:
```suggestion
    <string name="permission_explanation">Mir kennen den Hond leider net gesinn wann\'s du eis net erlaabs d\'Kamera ze benotzen.</string>
```

Apostrophes need to be escaped so that they don't get mixed with the xml syntax :wink:

Reply:
Whoops, right :)